core/primitives/src/version.rs

```diff
@@ -238,6 +244,7 @@ impl ProtocolFeature {
             ProtocolFeature::Ed25519Verify
             | ProtocolFeature::ZeroBalanceAccount
             | ProtocolFeature::DelegateAction => 59,
+            ProtocolFeature::NearVm => 60,
```

Comment:
As far as I understand testing on nightly would not actually test any new code paths, due to the fact that the running protocol version on both mainnet and testnet would not be near-vm anyway, but I may very well be wrong and will let @jakmeier comment :)

Reply:
Yes, betanet picking the feature up before we stabilize it _might_ be useful but not sure how much we gain from it.

> Is it truly alright to have this insta-stable? cc @jakmeier

Good question. I think it's more of a "common practice" that we 1) introduce a feature as unstable, 2) get everything in place, 3) make it stable. This process allows to partially merge changes without it affecting the stable version.
If we were to use e.g. a feature-branch approach, I don't think we would need to introduce unstable features in the first place.

IMO, this change seems like a special case because we already merged code that was protected by the VM selection (as opposed to using an unstable feature flag). Hence it seems find to make it insta stable.

But if this PR effectively stabilizes the new VM, we should add an entry in the changelog.

Reply:
Oh wait, I just noticed, isn't protocol version 60 already on testnet? We definitely shouldn't add features to versions that were already published. Even if we are okay with insta-stabilization, we need to bump to version 61 and use that here. (There might be a race condition with other PRs also bumping to 61)

Reply:
Ooh you’re right, thank you for the notice! I hadn’t noticed that since I started working on this PR set, testnet had time to go from being at version 58 to being at version 60 😅